### PR TITLE
Created readme

### PR DESCRIPTION
**Pull request by nikx1015**
This readme should tell a user how to:
-setup a database
-clone the project
-install dependencies
-edit the appsettings file
-launch the app
-navigate the app
-use the features of the app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Once this feature is implemented, it should work similarly to the features listed above.
+
+```TECH USED```
+
+C#, SQL
+
+```COLLABORATORS```
+
+This app was built in collaboration with Matt Rowe, Nikki Ash, Dale Saul, Russell Miller, and Josh Joseph.
```
Comment (ColonelSped) [inline README.md]: Please remove my name if you do not mind. :) 
Thank you

File: README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+If you have not already set up a database for this, you'll need to do so. To do this, open SSMS and click the option to create a new database. Name your database Ravioli and begin the process of setting this up. You can find all of the information you'll need here:
+https://gist.github.com/nikx1015/cb23e425bc71e489ef4017794537ea9a
+
+You can copy the code beginning with CREATE Table and ending before the ALTER table.
+Once you run this, your initial tables and columns should be set up and you can move on. Perform a new query with only the 3 lines following the ALTER command. Run this and then copy the code beneath that and post it into the query. Once you run that query, check to make sure your database is seeded properly before moving on.
+
+``CLONING THE GITHUB PROJECT``
+
+Next, you'll need to clone the files from Github and launch Visual Studio. Open the sln file you just created.
```
Comment (ColonelSped) [inline README.md]: .sln

File: README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+To create a new employee, click the affordance that says create new employee. You'll be redirected to a form where you may enter an employees information. Once you're finished, click submit and you should be redirected back to the employee list. Scroll down and your newly added employee should be there.
+
+To view an employees details, choose an employee from the employee list and click the details link beside their name. You'll be redirected to a new page with that employees information.
+
+``DEPARTMENTS``
+
+The Department feature of the app is functioning as well. You may view all departments or view a single department and the details of that department.
+
+TO use the features, use the ctrl+F5 command in visual studio to launch the app. Once it's loaded, you may click on the department affordance to be taken to the department section of the app. You should see a list of departments with an toption to view the details of each department. You should also see an affordance to create a new department.
```
Comment (ColonelSped) [inline README.md]: "To"

Comment (ColonelSped) [inline README.md]: "toption" or "to option"??

File: README.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+# bangazon-workforce-radvioli
+
+```BANGAZON WORKFORCE MANAGEMENT SITE```
+
+Bangazon Workforce Management Site is an internal application built for Human Resources and the IT Department. This application allows a user to create, list, and view Employees, Training Programs, Departments, and Computers.
+
+``DATABASE SETUP``
+
+If you have not already set up a database for this, you'll need to do so. To do this, open SSMS and click the option to create a new database. Name your database Ravioli and begin the process of setting this up. You can find all of the information you'll need here:
```
Comment (ColonelSped) [inline README.md]: What is SSMS?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+You can copy the code beginning with CREATE Table and ending before the ALTER table.
+Once you run this, your initial tables and columns should be set up and you can move on. Perform a new query with only the 3 lines following the ALTER command. Run this and then copy the code beneath that and post it into the query. Once you run that query, check to make sure your database is seeded properly before moving on.
+
+``CLONING THE GITHUB PROJECT``
+
+Next, you'll need to clone the files from Github and launch Visual Studio. Open the sln file you just created.
+
+``EDIT THE APPSETTINGS FILE``
+
+Go to your appsettings.json file and replace the code inside with this: { "Logging": { "LogLevel": { "Default": "Warning" } }, "AllowedHosts": "*", "ConnectionStrings": { "DefaultConnection": "Server=localhost\SQLEXPRESS;Database=Ravioli;Trusted_Connection=True;" } }
```
Comment (ColonelSped) [inline README.md:23]: Add "Instructions with using SQL Server" at the beginning of this.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+``DEPARTMENTS``
+
+The Department feature of the app is functioning as well. You may view all departments or view a single department and the details of that department.
+
+TO use the features, use the ctrl+F5 command in visual studio to launch the app. Once it's loaded, you may click on the department affordance to be taken to the department section of the app. You should see a list of departments with an toption to view the details of each department. You should also see an affordance to create a new department.
+
+To create a new department, click on the affordance that says create new department. You'll be redirected to a form where you may enter a departments information. Once you've finished, click submit and you should be redirected back to the list of departments. If you scroll to the bottom of this list, you should see the department you created.
+
+To view a departments details, choose a department from the department list and lick the details link beside the name of the department. You'll be redirected to a new page with that departments information.
```
Comment (ColonelSped) [inline README.md]: "lick" or "click" ?

**Review by ColonelSped — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (MattRowee) [reply README.md]: 😋

File: README.md (hunk 1 above)
Comment (nikx1015) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (nikx1015) [reply README.md]: also done

File: README.md (hunk 3 above)
Comment (nikx1015) [reply README.md]: done

Comment (nikx1015) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (nikx1015) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (nikx1015) [reply README.md:23]: done

File: README.md (hunk 6 above)
Comment (nikx1015) [reply README.md]: done

**Comment by MattRowee** [comment]
Didnt even have to read this one to know that its PERFECT 👍 💯 🥇

**Pull request merged**